### PR TITLE
Added 1000-SR.1 adoc files

### PR DESCRIPTION
**Pull request by manuelfuenmayor**
Hi Ron,
This time, I set `service-publication` as doctype. Please, check the attribute values. I'll be aware of your corrections.
Thanks!

### CODE REVIEW

File: 1000-SR.1/T-SP-SR.1-2012-MSW-E.adoc [hunk 1]
```diff
@@ -0,0 +1,519 @@
...
+
+a|*_Panama_* +
+*_Panama_* +
+*_Panamá_* +
+Collect calls to mobile numbers  are not accepted
+
+a|*_Papouasie-Nouvelle-Guinée_* +
+*_Papua New Guinea_* +
+*_Papua Nueva Guinea_* +
+Collect calls to mobile numbers  are not accepted +
```
Comment (ronaldtse) [inline 1000-SR.1/T-SP-SR.1-2012-MSW-E.adoc:341]: Double spaces should be made single.

**Review by ronaldtse — CHANGES_REQUESTED** [review]

**Comment by ronaldtse** [comment]
Everything else is good!

File: 1000-SR.1/T-SP-SR.1-2012-MSW-E.adoc (hunk 1 above)
Comment (manuelfuenmayor) [reply 1000-SR.1/T-SP-SR.1-2012-MSW-E.adoc:341]: Excuse me @ronaldtse , I don't see any double spaces. Could you please indicate me where are they?

**Comment by ronaldtse** [comment]
LGTM. Merging. Thanks @manuel489 !

**Review by ronaldtse — APPROVED** [review]

**Comment by ronaldtse** [comment]
@manuel489 just FYI, I nearly forgot -- the extracted data should be all pushed to https://github.com/ituob/service-publications-data/ . I've already moved all of them there. Thanks!

**Pull request merged**